src/core/agent/services/identifierService.test.ts

```diff
@@ -510,6 +510,7 @@ describe("Single sig service of agent", () => {
         isDeleted: true,
       }
     );
+    expect(updateIdentifierMock).toBeCalledTimes(1)
```

Comment:
also this isn't testing what we're updating to. This isn't specific enough. Let me push a commit to get this merged.